sympy/combinatorics/tests/test_schur_number.py

```diff
@@ -0,0 +1,31 @@
+from sympy.combinatorics.schur_number import schur_partition
+import random
+
+"""
+Checks if every subset is sum-free(There are no x,y,z in the subset such that
+x + y = z)
+"""
+def _sum_free_test(subset):
+    for i in subset:
+        for j in subset:
+            if i + j in subset:
+                raise AssertionError("This subset is not sum free")
+    return "Everything is correct"
+
+
+n = random.randint(1,1000)
+result = schur_partition(n)
+
+"""
+Checks if the occurance of all numbers  is exactly one
+"""
+t = 0
```

Comment:
Tests should go inside of a test function. See how the other test files in SymPy are organized.